File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/op/atts/RecordAttsDao.java
package ru.citeck.ecos.records3.record.op.atts;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;
import ru.citeck.ecos.records3.record.dao.RecordsDao;

public interface RecordAttsDao extends RecordsDao {

    @Nullable
    Object getRecordAtts(@NotNull String record);
}

File: ecos-records/src/main/java/ru/citeck/ecos/records3/record/dao/atts/RecordAttsDao.kt
package ru.citeck.ecos.records3.record.dao.atts

import ru.citeck.ecos.records3.record.dao.RecordsDao
import kotlin.jvm.Throws

interface RecordAttsDao : RecordsDao {

    /**
     * It is adapter to RecordsAttsDao for single record
     * @see RecordsAttsDao
     */
    @Throws(Exception::class)
    fun getRecordAtts(recordId: String): Any?
}
